apps/system/js/app_window_factory.js

```diff
@@ -218,11 +218,7 @@
           this.publish('launchapp', config);
         };
 
-        if (Service.query('MultiScreenController.enabled')) {
-          Service.request('chooseDisplay', config).catch(launchApp);
-        } else {
-          launchApp();
-        }
+        launchApp();
```

Comment:
The function `launchApp` was created for the `if / else` case only. Since the case no longer exists, I would suggest you unwrap the `launchApp` function to make the code simpler.